client/src/misc.ts

```diff
@@ -2,6 +2,8 @@
 // NOTE: This is because this file introduces a generic extension for
 // all JQuery objects
 
+import globals from './game/ui/globals';
```

Comment:
misc probably shouldn't be importing globals. how about we move `resetActiveHover()` to `client/src/game/ui/tooltips.ts`